controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -421,45 +442,39 @@ public void authTokenCheckFailed(WireCommands.AuthTokenCheckFailed authTokenChec
                                                          final UUID txId,
                                                          final HostControllerStore hostControllerStore,
                                                          final ConnectionFactory clientCF, String delegationToken) {
-        final String qualifiedName = StreamSegmentNameUtils.getQualifiedStreamSegmentName(scope, stream, segmentId);
+        final String transactionName = getTransactionName(scope, stream, segmentId, txId);
         final Controller.NodeUri uri = getSegmentUri(scope, stream, segmentId, hostControllerStore);
         final CompletableFuture<TxnStatus> result = new CompletableFuture<>();
-        final WireCommandType type = WireCommandType.ABORT_TRANSACTION;
+        final WireCommandType type = WireCommandType.DELETE_SEGMENT;
         final FailingReplyProcessor replyProcessor = new FailingReplyProcessor() {
 
             @Override
             public void connectionDropped() {
-                log.warn("abortTransaction {} connectionDropped", qualifiedName);
+                log.warn("abortTransaction {} connectionDropped", transactionName);
                 result.completeExceptionally(new WireCommandFailedException(type, WireCommandFailedException.Reason.ConnectionDropped));
             }
 
             @Override
             public void wrongHost(WireCommands.WrongHost wrongHost) {
-                log.warn("abortTransaction {} wrongHost", qualifiedName);
+                log.warn("abortTransaction {} wrongHost", transactionName);
                 result.completeExceptionally(new WireCommandFailedException(type, WireCommandFailedException.Reason.UnknownHost));
             }
 
             @Override
-            public void transactionCommitted(WireCommands.TransactionCommitted transactionCommitted) {
-                log.warn("abortTransaction {} TransactionCommitted", qualifiedName);
-                result.completeExceptionally(new WireCommandFailedException(type, WireCommandFailedException.Reason.PreconditionFailed));
-            }
-
-            @Override
-            public void transactionAborted(WireCommands.TransactionAborted transactionDropped) {
-                log.debug("abortTransaction {} transactionAborted", qualifiedName);
+            public void segmentDeleted(WireCommands.SegmentDeleted transactionAborted) {
+                log.debug("abortTransaction {} transactionAborted", transactionName);
```

Comment:
log message

Reply:
this is correct. because we have removed AbortTransaction wireCommand and instead we send DELETE_SEGMENT. So SegmentDeleted is the expected happy day response. and since controller has called it in the context of abort transaction, so it says transactionAborted